.github/workflows/examples-openiotsdk.yaml

```diff
@@ -127,8 +127,8 @@ jobs:
                   scripts/run_in_ns.sh ${TEST_NETWORK_NAME}ns scripts/examples/openiotsdk_example.sh -C test -n ${TEST_NETWORK_NAME}tap lock-app
                   scripts/setup/openiotsdk/network_setup.sh -n $TEST_NETWORK_NAME down
             
-            - name: Run unit tests
+            - name: Test unit-tests
```

Comment:
This is a bit awkward, I suggest to add a colon between `Test` and the app being tested: `Test: unit-tests`. This can be applied to other test steps